### PR TITLE
Bump ImageMagick, build for experimental platforms

### PR DESCRIPTION
**Pull request by staticfloat**

### CODE REVIEW

File: I/ImageMagick/build_tarballs.jl [hunk 1]
```diff
@@ -2,12 +2,12 @@
 # `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder
 name = "ImageMagick"
-version = v"6.9.10-12"
+version = v"6.9.12-19"
```
Comment (giordano) [inline I/ImageMagick/build_tarballs.jl:5]: Can we not use the prerelease tag?

**Comment by giordano** [comment]
Windows is chocking on
```
[18:50:07] magick/timer.c:205:20: error: ‘CLOCK_REALTIME’ undeclared (first use in this function)
[18:50:07]  #  define CLOCK_ID CLOCK_REALTIME
[18:50:07]                     ^
[18:50:07] magick/timer.c:211:24: note: in expansion of macro ‘CLOCK_ID’
```
See https://github.com/ImageMagick/ImageMagick6/blob/1e43b291a5825520860b92acb3f7b52ba6145805/magick/timer.c#L205.  This line was added 5 years ago.  I can't find `CLOCK_REALTIME` anywhere in the target header files.  How did this ever work?

**Comment by anandijain** [comment]
whats the status on this one? it'd be nice to be able to use WGLMakie

**Comment by aviks** [comment]
Creating an explicit link: Apparently, this version of the jll breaks the wrapper package https://github.com/JuliaIO/ImageMagick.jl/issues/206

**Pull request merged**